corehq/apps/case_importer/templates/case_importer/partials/ko_import_status.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+<!--ko if: state === $root.states.SUCCESS-->
+    <div class="text-success">
+        <!--ko if: result.match_count-->
+            <p>
+                {% blocktrans with match_count='result.match_count' %}
+                    <strong data-bind="text: {{ match_count }}"></strong> rows were matched and
+                    updated.
+                {% endblocktrans %}
+            </p>
+        <!--/ko-->
+
+        <!--ko if: result.created_count > 0-->
+            <p>
+                {% blocktrans with created_count='result.created_count' %}
```

Comment:
Sorry this is confusing. This line sets the django template variable `created_count` to the string `'result.created_count'`, which is then used below. The point was just remove as much logic as possible from the thing inside blocktrans, but it's still pretty awkward. Not sure if there's a much better way to do this.